themes/default/content/request-a-demo/_index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Request a demo
+meta_desc: Request a demo with a cloud engineering expert to see how Pulumi can help you ship infrastructure faster.
+
+type: page
+layout: demo
+
+overview:
+    title: See how Pulumi can help you ship infrastructure faster.
+    details:
```

Comment:
For these three I would make it even more aggressive. Just some feedback on the bullet points. The first one is too nebulous with "challenges you are facing today" because most companies have a ton. The second bullet point is a more fringe use case that is alluding to maybe SSPs or custom CLI tooling. I would make the third one more punchy